### PR TITLE
Feature phone field local

### PR DESCRIPTION
**Pull request by thomas-cleany**

### CODE REVIEW

File: docs/Fields.md [hunk 1]
```diff
@@ -283,6 +283,34 @@ See [Intl.Numberformat documentation](https://developer.mozilla.org/en-US/docs/W
 
 **Tip**: If you need more formatting options than what `Intl.Numberformat` can provide, build your own field component leveraging a third-party library like [numeral.js](http://numeraljs.com/).
 
+## `<PhoneField>`
+
+`<PhoneField>` displays a phone number as a `<a href="tel:" />` link.
+
+If the number is from the locale it'll be formatted like a national number, if not international formatting will be set.
```
Comment (sebastien-cleany) [inline docs/Fields.md]: You could add that the component accepts local and international phone number as long as the locale is the one set in the props.

File: example/comments.js [hunk 2]
```diff
@@ -72,7 +73,9 @@ const CommentGrid = translate(({ ids, data, basePath, translate }) => (
         <Card key={id} style={cardStyle}>
             <CardHeader
                 title={<TextField record={data[id]} source="author.name" />}
-                subtitle={<DateField record={data[id]} source="created_at" />}
+		subtitle={
```
Comment (sebastien-cleany) [inline example/comments.js]: weird indent

File: src/mui/field/PhoneField.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import get from 'lodash.get';
+import pure from 'recompose/pure';
+
+import { parse, format } from 'libphonenumber-js';
+
+export const localPhoneFormat = (input, locale) => {
+  if (!locale) {
```
Comment (sebastien-cleany) [inline src/mui/field/PhoneField.js]: indent in admin-on-rest use 4 spaces.

File: src/mui/field/PhoneField.js [hunk 4]
```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import get from 'lodash.get';
+import pure from 'recompose/pure';
+
+import { parse, format } from 'libphonenumber-js';
+
+export const localPhoneFormat = (input, locale) => {
```
Comment (sebastien-cleany) [inline src/mui/field/PhoneField.js]: avoid meaningless variable name as input/output/string/...
parsed_number would be nice in this case.

File: src/mui/field/PhoneField.js [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+  return input;
+};
+
+const PhoneField = ({ source, record, elStyle, locale }) => {
+  if (!record) {
+    return null;
+  }
+
+  const parsed_number = parse(get(record, source), {country: {default: locale}});
+  console.log(parsed_number);
```
Comment (sebastien-cleany) [inline src/mui/field/PhoneField.js]: forgotten console.log

File: example/comments.js [hunk 6]
```diff
@@ -72,7 +73,9 @@ const CommentGrid = translate(({ ids, data, basePath, translate }) => (
...
+		    <PhoneField record={data[id]} source="author.phone" />
```
Comment (philippe-cleany) [inline example/comments.js]: on one line

File: src/mui/field/PhoneField.spec.js [hunk 7]
```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import assert from 'assert';
+import { shallow } from 'enzyme';
+import PhoneField, { localPhoneFormat } from './PhoneField';
+
+describe.only('<PhoneField />', () => {
+  // localPhoneFormat tests
```
Comment (sebastien-cleany) [inline src/mui/field/PhoneField.spec.js]: instead of having useless comments you could separate test cases in 2 different describes :)

**Pull request merged**